parsers/cxx/cxx_parser_template.c

```diff
@@ -421,35 +424,8 @@ cxxParserParseTemplateAngleBracketsInternal(bool bCaptureTypeParameters,int iNes
 
 				CXX_DEBUG_PRINT("Found %d greater-than signs",iGreaterThanCount);
 
-				// check greater than operator: very narrow conditions
-				if(
-					(iGreaterThanCount == 1) &&
-					(
-						// whatever op 2 [C++03 allows this without parens]
-						// whatever op (...) [C++03 allows this without parens]
-						cxxTokenTypeIsOneOf(
-								g_cxx.pToken,
-								CXXTokenTypeNumber | CXXTokenTypeOpeningParenthesis
-							) ||
-						// whatever op nonTypeParameter [C++03 allows this without parens]
-						(
-							cxxTokenTypeIs(g_cxx.pToken,CXXTokenTypeIdentifier) &&
-							cxxTokenIsPresentInTemplateParametersAsNonType(g_cxx.pToken)
-						)
-						// WARNING: don't be tempted to add a loose condition that has
-						// (!cxxTokenIsPresentInTemplateParametersAsType()) on the right.
-						// It's unsafe.
-					)
-				)
-				{
-					CXX_DEBUG_PRINT("Treating as greater-than sign");
-
-					if(cxxTokenTypeIsOneOf(g_cxx.pToken,CXXTokenTypeOpeningParenthesis))
-						cxxParserUngetCurrentToken(); // needs to be condensed
-
-					continue;
-				}
-
+                // We do not check for the greater than operator, as it is only valid
```

Comment:
A minor request.
Could you use "tab" characters instead of space characters for indentation?

Reply:
✅